File: C++/fibonacci.cpp
#include<bits/stdc++.h>
using namespace std;
int fib(int n){
    if(n==1){
        return 0;
    }
    if(n==2){
        return 1;
    }
    if(n>2){
        return fib(n-1)+fib(n-2);
    }
    return 0;
}
int main(){
    int n;
    cout<<"Enter the number: ";
    cin>>n;
    cout<<"Fibonacci series till the given term is"<<endl;
    for(int i=1;i<=n;i++){
        cout<<fib(i)<<" ";
    }
    return 0;
}

File: Python/Fibonacci.py
n=int(input("Enter number of terms:"))
a, b=0, 1
c=0
if n<=0:
 print("Enter positive number of terms")
elif n==1:
 print("Fibonacci series upto",n,":",a)
else:
 for i in range (c,n):
      print(a)
      sum=a+b
      a=b
      b=sum
      c=c+1
